sdk/communication/AzureCommunicationChat/Tests/Recordings/.dat.nosync108b.AjWTwN

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
What is up with this file name? We should remove this and see if the tests still pass.

Reply:
This file doesn't show up explicitly in my local repo but looks that it is generated automatically in git.
I just removed it and the playback tests still pass.

Reply:
Run the project again and didn't see the file with such weird name anymore.